src-java/blue-green/src/main/java/org/openkilda/bluegreen/kafka/interceptors/VersioningConsumerInterceptor.java

```diff
@@ -61,6 +63,12 @@ public VersioningConsumerInterceptor() {
             for (ConsumerRecord<K, V> record : records.records(partition)) {
                 if (checkRecordVersion(record)) {
                     filteredRecords.add(record);
+
+                    if (record.value() instanceof DeserializationError) {
```

Comment:
Why are you analyzing data here?

If this record have passed `checkRecordVersion()` it will be seen to main code and it will / should react property. You do not report anything if record do not pass `checkRecordVersion()`... I can't say right now is it correct or not.

Reply:
let me describe the problem. We have 2 versions of kilda: blue and green.
For example we added/deleted/changed new field in some message in green version.

Here is the problem: kafka firstly deserialize a message and only then run interceptor and check version. 
it mean that messages with different kilda will log deserialization error. 
How we can fix it? just by changing log level because such messaged are false posotive.

But what if we really have a deserialization problem inside one Kilda version. For example green network sent to green router some invalid message? we will just miss error log.
So I added this analyzation here to log error ONLY for messaged with right kafka version. Because messages with wrong kafka version will have false positive errors.